cloud data import: update data import docs as per new UI (#10265)
pingcap/docs: tidb-cloud/migrate-from-amazon-s3-or-gcs.md

### Step 3. Import data into TiDB Cloud

1. Log in to the [TiDB Cloud console](https://tidbcloud.com/), and navigate to the **Clusters** page.

2. Locate your target cluster, and click **Import Data** in the upper-right corner of the cluster area. The **Data Import** page is displayed.

3. On the **Data Import** page, fill in the following information:

    - **Data Format**: choose the format of your data.
    - **Location**: `AWS`
    - **Bucket URL**: fill in the bucket URL of your source data.
    - **Role-ARN**: enter the Role-ARN you obtained in [Step 2](#step-2-configure-amazon-s3-access).

>     - **Target Cluster**: fill in the **Username** and **Password** fields.

Author: ```suggestion
    - **Target Cluster**: shows the cluster name and the region name.
```

>     Confirm the compliance of cross region. Click **Next**.

Author: ```suggestion
    Confirm the compliance of cross region. Click **Next**.
    
    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

>     If the location of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.

Reviewer: ditto

>     TiDB Cloud starts validating whether it can access your data in the specified bucket URL. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).

Author: ```suggestion
    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After validation, TiDB Cloud tries to scan all the files in the data source using the default file naming pattern, and returns a scan summary result on the left side of the next page. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

> 4. Modify the file patterns and add the table filter rules if needed. If necessary, you can specify a [table filter](/table-filter.md#syntax). If you want to configure multiple filter rules, use `,` to separate the rules.

Reviewer: ditto for multiple filter rules

> 4. Modify the file patterns and add the table filter rules if needed. If necessary, you can specify a [table filter](/table-filter.md#syntax). If you want to configure multiple filter rules, use `,` to separate the rules.

Author: ```suggestion
4. Modify the file patterns and add the table filter rules if needed.
```

6. On the **Preview** page, confirm the import data and then click **Start Import**.

> TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).

Reviewer: "TiDB Cloud starts validating whether it can access your data in the specified bucket URL", "If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md)."
ditto

> TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).

Author: ```suggestion
```

### Step 3. Import data into TiDB Cloud

1. Log in to the [TiDB Cloud console](https://tidbcloud.com/), and navigate to the **Clusters** page.

> 2. Locate your target cluster, and click **Import Data** in the upper-right corner of the cluster area. The **Data Import** page is displayed.

Author: ```suggestion
2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed..
```

        gsutil rsync -r ./tidbcloud-samples-us-west-2/ gs://target-url-in-gcs
        ```

> 2. From the TiDB Cloud console, navigate to the **Clusters** page, and then click the name of your target cluster to go to its own overview page. In the **Import** area, click **Import Data**, and then fill in the importing related information on the **Data Import** page.

Author: ```suggestion
2. From the TiDB Cloud console, navigate to the **Clusters** page, and then click the name of your target cluster to go to its own overview page. Click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed. Fill in the importing related information. 
```
